### ISSUE TITLE
New Grayscale detection algorithm

### ISSUE THREAD

**Issue by local-minimum** [Scan-o-Matic/scanomatic-standalone#33]
# 📋 Info
General problems with identifying grayscale, selecting grayscale when identifying a fixture

We need to find a more durable option for gray scale. The current method allows it to accumulate dust and degrade rather quickly. Moreover, if somehow the grayscale moved / tilted slightly in its place, it becomes impossible to analyse that project data without gray scale hacking. 

# Requirements
- As many as images possible with troublesome GS detection from as many different projects and fixtures as posible
- As many as images possible with deteriorated GS from as many different projects and fixtures as posible

# 🏁 DoD

- [ ] Implement more robust GS detection

**Comment by dodslaser** [comment]
Grayscale detection has been a major pain-point for us at GU, to the point where people are habitually hacking old grayscale values into their scans.

I know this has been brought up in the past, but we would like to have to bypass this somehow. One option would be to run calibration routinely (e.g. on a schedule, or before each project) and then use those values, as this is essentially what we have been doing anyway. As far as we can tell, the effect is minimal. Preferably, we would like to run calibration on an unmodified IT8 target of some format.

**Comment by local-minimum** [comment]
I agree, the main need for this calibration was when the scanners didn't use LED lamps. The LED light source seems much more stable.

It could be possible to make a "no-grascale" grayscale detection that just outputs the same values no matter the image and then there's no need to touch the other parts of the project since they will believe that they detected a grayscale like they've always done.
